src/api-doc/api-doc.yaml

```diff
@@ -4959,6 +4959,696 @@ paths:
+         - The API Endpoint for create rollout is `/scp/v1/rollouts/update` 
+         - It is mandatory to provide values for parameters which are marked as `required` 
+         - This is a mandatory parameter and cannot be empty or null.
+      parameters:
+        - name: X-auth-token
+          in: header
+          description: >-
+            To use this API, you require an X-auth-token. This is available in
+            the login API Response.
+          required: true
+          schema:
+            type: string
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                title:
+                  type: string
+                  example: test rollout
+                resource_id:
+                  type: number
+                  example: 2
+                targeting_criteria:
+                  type: array
+                  items:
+                    type: object
+                    properties:
+                      state:
+                        type: string
+                        example: 6687b8d38ead9320cf997c65
+                      entity_targeting:
+                        type: string
+                        example: district
+                      district:
+                        type: array
+                        items:
+                          type: string
+                          example: 671097d667b6747799a761a4
+                      block:
+                        type: array
+                        items:
+                          type: string
+                          example: 6710d01167b6747799a7671d
+                      gender:
+                        type: array
+                        items:
+                          type: string
+                          example: male
+                      roles:
+                        type: array
+                        items:
+                          type: string
+                          example: deo
+                viewers:
+                  type: array
+                  items:
+                    type: number
+                    example: 23
+                start_date:
+                  type: string
+                  format: date-time
+                  example: 2024-10-29T11:35:08.694Z
+                end_date:
+                  type: string
+                  format: date-time
+                  example: 2024-11-29T11:36:31.117Z
+
+      responses:
+        '201':
+          description: Changes published successfully
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  responseCode:
+                    type: string
+                    example: OK
+                  message:
+                    type: string
+                    example: Your changes have been published
```

Comment:
Can we have this message "Your changes have been saved" ? As we have a different API for publish? @priyanka-TL

Reply:
It's the message provided by product